src/components/DimensionsPanel/Dialogs/DialogManager.js

```diff
@@ -553,7 +554,7 @@ DialogManager.propTypes = {
     metadata: PropTypes.object,
     parentGraphMap: PropTypes.object,
     rootOrgUnits: PropTypes.array,
-    selectedItems: PropTypes.object,
+    selectedItems: PropTypes.func,
```

Comment:
Ideally this would have a name that suggests it is a function instead of an array